components/Account/Access/Campaigns/Form.js

```diff
@@ -0,0 +1,168 @@
+import React, { Component, Fragment } from 'react'
+import isEmail from 'validator/lib/isEmail'
+
+import { Button, Field, Label, InlineSpinner, Interaction, A } from '@project-r/styleguide'
+
+import { timeFormat } from '../../../../lib/utils/format'
+
+import ErrorMessage from '../../../ErrorMessage'
+
+const { H3, P } = Interaction
+
+const dayFormat = timeFormat('%e. %B %Y')
+
+class Form extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      email: '',
+      error: false,
+      isMutating: false,
+      hasMutated: false,
+      mutationError: null,
+      dirty: {},
+      grantedLast: {}
+    }
+
+    this.hasMutated = ({ data }) => {
+      this.setState({
+        isMutating: false,
+        hasMutated: true,
+        grantedLast: {
+          email: data.grantAccess.email,
+          endAt: data.grantAccess.endAt
+        }
+      })
+    }
+
+    this.catchMutationError = error => {
+      this.setState({
+        isMutating: false,
+        mutationError: error
+      })
+    }
+
+    this.onClickReset = (event, value, shouldValidate) => {
+      event.preventDefault()
+
+      this.setState({
+        value: '',
+        hasMutated: false,
+        dirty: {}
+      })
+    }
+
+    this.onChangeEmail = (event, value, shouldValidate) => {
+      event.preventDefault()
+
+      this.setState(() => ({
+        value,
+        error: (
+          value.trim().length <= 0 &&
+          'Keine E-Mail-Adresse angegeben'
```

Comment:
Make sure to use `t`. Also consider that the error replaced the label, I prefer following style when possible:  
`E-Mail-Adresse`
`E-Mail-Adresse fehlt`  
`E-Mail-Adresse ungültig`  
_keeping the label in the same place at the beginning_